README.md

```diff
@@ -481,6 +481,10 @@ By default this component adds a parameter to the generated url to help imgix wi
 
 Any other attributes to add to the html node (example: `alt`, `data-*`, `className`).
 
+##### targetMaxWidth :: number
+
+The maximum size for the auto generated `srcSet`.
```

Comment:
```suggestion
By default this library generates srcset widths up to 8192px wide. This can be reduced by setting `targetMaxWidth` to a number lower than 8192, e.g. `targetMaxWidth={4000}`.
```